test/keychange_listener_test.js

```diff
@@ -1,5 +1,6 @@
 describe('KeyChangeListener', function() {
   var phoneNumberWithKeyChange = '+13016886524';  // nsa
+  var address = new libsignal.SignalProtocolAddress(phoneNumberWithKeyChange, 1);
```

Comment:
Ah, I see what happened. This branch was previously on top of the `session-archive` branch, and when I rebased straight back on top of `combined`, this line was orphaned. It was replacing a line which caused a crash because it was referencing a nonexistent `identifier` variable, if I recall correctly.